sdk/metric/periodic_reader.go

```diff
@@ -68,7 +68,7 @@ func (o periodicReaderOptionFunc) applyPeriodic(conf periodicReaderConfig) perio
 
 // WithTimeout configures the time a PeriodicReader waits for an export to
 // complete before canceling it. This includes an export which occurs as part
-// of Shutdown.
+// of ForceFlush and Shutdown.
```

Comment:
I'm mostly OK with this, as I think most users will understand what we mean... But technically ForceFlush and Export are different methods on an exporter.  "an export which occurs as part of ForceFlush" might be confusing if ForceFlush does more than export (or doesn't export?).

Reply:
https://github.com/open-telemetry/opentelemetry-go/blob/c30a8f7649b62201a7782cbeb5fc4a890bfd56f6/sdk/metric/periodic_reader.go#L106-L110

+

https://github.com/open-telemetry/opentelemetry-go/blob/c30a8f7649b62201a7782cbeb5fc4a890bfd56f6/sdk/metric/periodic_reader.go#L297-L299

I think we should improve the doc comment of `ForceFlush` (seperate PR please 😉 ).

Reply:
The comment was already changed in https://github.com/open-telemetry/opentelemetry-go/pull/4377.

Reply:
This looks to be resolved.